homework/svitlana-kalmutska/lesson10/hw_1/css/styles.css

```diff
@@ -0,0 +1,195 @@
+* {
+    box-sizing: border-box;
+}
+
+@font-face {
+    font-family: 'Raleway-Light';
+    src: url(/fonts/Raleway-Light.woff2) format('woff2'),
+        url(/fonts/Raleway-Light.woff) format('woff');
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Raleway-Bold';
+    src: url(/fonts/Raleway-Bold.woff2) format('woff2'),
+        url(/fonts/Raleway-Bold.woff) format('woff');
+    font-style: normal;
+    font-display: swap;
+}
+
+@font-face {
+    font-family: 'Raleway-Black';
+    src: url(/fonts/Raleway-Black.woff2) format('woff2'),
+        url(/fonts/Raleway-Black.woff) format('woff');
+    font-style: normal;
+    font-display: swap;
+}
+
+.container {
+    max-width: 1500px;
+}
+
+.header {
+    background-image: url(/images/bg.jpg);
+    height: 1087px;
+    max-width: 1500px;
+}
+
+/* I made one more div (header__container) because I desided not to write margin in block .header as it could be used in future like separate block. And I also can't write margins in .container, because in that case I have problems with my backgroun images (our header should be 1500 px max-width because we have such bg-img) */
+.header__container {
```

Comment:
it is a bad practice to use so familiar names for classes like .header-container and .header__container